src/components/qas/questions/qaCalculators/sizeCalculator/index.jsx

```diff
@@ -0,0 +1,242 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { CardText, CardActions } from 'react-md';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import classnames from 'classnames';
+// import Calculation from '../shared/calculation';
+import CalculatedMeasurement from '../shared/CalculatedMeasurement';
+import Equation from '../shared/equation';
+import TextField from '../../../../site/forms/textField';
+import Card from '../../../../site/card';
+import { renderDef, formatValue } from '../../../../../lib/utilities.js';
+import './sizeCalculator.module.scss';
+
+const HIcon = () => {
+  return (
+    <div>
+      <strong>H</strong> =
+    </div>
+  );
+};
+
+const PIcon = () => {
+  return (
+    <div>
+      <strong>p</strong> =
+    </div>
+  );
+};
```

Comment:
I'm getting some funny
```
h
=
```
 text wrapping at narrow (but still reasonably wide) screen sizes.  I think it's because the `h` and the `=` are both `display: inline`.  I think you can fix by making them both inline-block (i.e. wrapping the = and whitespace in a span and setting it to inline-block, and updating display property of the strong).